django/db/models/query.py

```diff
@@ -1304,7 +1314,7 @@ def __getstate__(self):
         obj_dict = self.__dict__.copy()
         if self.queryset is not None:
             # Prevent the QuerySet from being evaluated
-            obj_dict['queryset'] = self.queryset._clone(
+            obj_dict['queryset'] = self.queryset._pre_next_op(
```

Comment:
Besides in `_chain()`, `_pre_next_op` is only called here so if we replaced this call with `_chain()`, the `_pre_next_opt` logic could be inlined in `_chain()`. Then we'd only have `QuerySet._chain()` and `_clone()` which might be simpler. Otherwise, I'm not sure how to distinguish the docstrings of  `_chain()` and `_pre_next_op()`.
`_clone()` might be: Create a copy of the current instance.
`_chain()` might be: Copy the QuerySet and prepare for the next QuerySet operation.
`_pre_next_op()` might be: Prepare for the next QuerySet operation without copying it.
But it seems uncertain how to determine which one to use.